### PR TITLE
gives milk very low brute healing

### PR DESCRIPTION
**Pull request by boodaliboo**
i thought for years milk healed brute but i was wrong,
this gives milk a 5% probability to heal 0.5 brute every second
:cl:  
rscadd: Added milk healing
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
> i thought for years milk healed brute but i was wrong, this gives milk a 5% probability to heal 0.5 brute every second 🆑 rscadd: Added milk healing /🆑

should i buff it actually because cream seems to heal better then this

**Comment by VaelophisNyx** [comment]
instead give cream double the healing on the same chance, but also make cream fill you up fast since it's like 20% fat

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -199,6 +199,12 @@
 	glass_name = "glass of milk"
 	glass_desc = "White and nutritious goodness!"
 
+/datum/reagent/consumable/milk/on_mob_life(mob/living/carbon/M)
+	if(M.getBruteLoss() && prob(5))
+		M.heal_bodypart_damage(0.5)
+		. = 1
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:205]:
```suggestion
		. = TRUE
```

**Comment by JamieD1** [comment]
Why the fuck does milk heal.

**Pull request closed without merging**